### PR TITLE
Make LiteDBStore & FileStore to atomically store transactions

### PR DESCRIPTION
**Pull request by dahlia**
This patch is an alternative solution to the same problem that #411 aims to address.

### CODE REVIEW

File: Libplanet.Tests/Store/StoreTest.cs [hunk 1]
```diff
@@ -542,5 +543,114 @@ public void IndexBlockHashReturnNull()
...
+            Task[] tasks = new Task[taskCount];
+            for (int i = 0; i < taskCount; i++)
+            {
+                var task = new Task(() =>
+                {
+                    PrivateKey key = new PrivateKey();
+                    var random = new System.Random();
+                    var md5 = MD5.Create();
+                    Transaction<AtomicityTestAction> tx;
+                    for (int j = 0; j < 50; j++)
```
Comment (earlbread) [inline Libplanet.Tests/Store/StoreTest.cs]: Shouldn't this be `txCount` too?

Comment (dahlia) [reply Libplanet.Tests/Store/StoreTest.cs]: Not necessarily; it's arbitrary.

File: Libplanet/Net/Swarm.cs [hunk 2]
```diff
@@ -1977,11 +1977,11 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
                 NetMQMessage raw = e.Socket.ReceiveMultipartMessage();
 
                 _logger.Verbose(
-                    "The raw message[frame count: {0}] has received.",
+                    "A raw message [frame count: {0}] has received.",
                     raw.FrameCount
                 );
                 Message message = Message.Parse(raw, reply: false);
-                _logger.Debug($"The message[{message}] has parsed.");
+                _logger.Debug($"A message has parsed: {0}", message);
```
Comment (longfin) [inline Libplanet/Net/Swarm.cs]:
```suggestion
                _logger.Debug("A message has parsed: {0}", message);
```

Comment (dahlia) [reply Libplanet/Net/Swarm.cs]: Amended!

**Comment by dahlia** [comment]
@earlbread @longfin Sorry, I rebased it on the current master again.

**Pull request merged**